tdrs-frontend/src/components/ResourceCards/ResourceCards.jsx

```diff
@@ -0,0 +1,128 @@
+import Button from '../Button'
+
+function ResourceCards() {
+  return (
+    <section className="padding-top-4 usa-section">
+      <div className="grid-row">
+        <div className="mobile:grid-container desktop:padding-0 desktop:grid-col-3">
+          <h2 className="resources-header font-heading-2xl margin-top-0 margin-bottom-0">
+            Featured TANF Resources
+          </h2>
+          <div>
+            <p>Questions about TANF data?</p>
+            <p>
+              Email:{' '}
+              <a className="usa-link" href="mailto: tanfdata@acf.hhs.gov">
+                tanfdata@acf.hhs.gov
+              </a>
+            </p>
+          </div>
+        </div>
+        <div className="desktop:grid-col-9">
+          <ul className="grid-row usa-card-group mobile:margin-0">
+            <li className="usa-card--header-first padding-bottom-4 desktop:padding-right-2 desktop:grid-col-6 mobile:grid-col-12">
+              <div className="usa-card__container">
+                <header className="usa-card__header">
+                  <h3 className="usa-card__heading">TDP Knowledge Center</h3>
+                </header>
+                <div className="usa-card__body">
+                  <p>
+                    The knowledge center contains resources on all things TDP
+                    from account creation to data submission.
+                  </p>
+                </div>
+                <div className="usa-card__footer">
+                  <Button
+                    type="button"
+                    class="usa-button"
+                    id="viewKnowledgeCenterButton"
+                    href="http://tdp-project-updates.app.cloud.gov/knowledge-center/"
+                  >
+                    View Knowledge Center
+                  </Button>
+                </div>
+              </div>
```

Comment:
Potentially turn this into generic card component that takes a title, description, link, and buttonText.